gnomad_constraint/utils/constraint.py

```diff
@@ -112,21 +112,28 @@ def prepare_ht_for_constraint_calculations(
     # collapse strands to deduplicate the context.
     ht = annotate_mutation_type(collapse_strand(ht))
 
+    # Obtain field name for median exome coverage.
```

Comment:
I know this is going to change anyway for v4, but as is it also won't work with the stratified coverage HT

Reply:
added todo